### DISCUSSION TITLE
🔦 Feature: option to share specific view to users instead all views

### DISCUSSION THREAD

**Discussion by PrasanthKumar77** [nocodb/nocodb#3882] [Q&A] [answered]
### Please confirm if feature request does NOT exists already ?

- [X] I confirm there is no existing issue for this

### Describe the usecase for the feature

i want to share specific views to my invited users instead of all views that are currently in a table. i can share a view but it i want to configure what view my invited users will see

### Suggested Solution

give a option to show all views are only selected views when giving a user viewer permission or inviting as a user. it wont apply for for other roles

**Comment by dstala** [comment, marked as answer]
@PrasanthKumar77 : at the role level, you can apply access control & limit what Editor/Commenter/Viewer can access.
You can open Project menu, go to `Team & Settings` > `Project Metadata` > `UI Access Control`

![Screenshot 2022-09-30 at 8 38 31 PM](https://user-images.githubusercontent.com/86527202/193301106-519543e0-41bb-4306-a099-7bfaa8ae9ab7.png)

**Comment by o1lab** [comment]
Moving this to discussion.

**Comment by PrasanthKumar77** [comment]
ohh! thanks @dstala  i missed it. will check it out.

**Reply by Shahin-rmz** [reply]
Hello, thanks for your answer I have also had a same problm.
2 questions though, I am using docker version and all I can see in `Team & Settings` is this page.
![image](https://github.com/nocodb/nocodb/assets/32805117/e9f00a58-f2dd-4436-8040-cf4a832eb710)


Q1:  Am I missing something?
which version is production ready?

---

Q2: There are scenarios, where I want to give one user with `editor` previlage, access to some views, and another users access to another views. how is that possible?

Thanks

**Reply by Sov3rain** [reply]
With the new versions of Noco, those settings seems to have disappeared. Am I right @dstala ? It's a real bummer, be able to filter who can view what was nice.
